modules/common/src/pubsub/consts/index.ts

```diff
@@ -0,0 +1,4 @@
+import ms = require('ms');
+
+export const SKIP_DEADLINE: number = ms('1s');
+export const DELAY_AFTER_STARTUP: number = ms('5s');
```

Comment:
shouldn't those 2 consts be still passed over and have those values as defaults?